test/v2/controllers/V2DeparturesControllerSpec.scala

```diff
@@ -1083,6 +1115,99 @@ class V2DeparturesControllerSpec
     }
   }
 
+  // Version 2
```

Comment:
what's this comment for? this entire spec is V2

Reply:
removed